src/main/java/baubles/common/items/ItemRing.java

```diff
@@ -18,33 +14,48 @@
 import net.minecraft.util.EnumHand;
 import net.minecraft.util.NonNullList;
 import net.minecraft.world.World;
+import net.minecraftforge.event.RegistryEvent;
+import net.minecraftforge.fml.common.Mod;
+import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.relauncher.Side;
 import net.minecraftforge.fml.relauncher.SideOnly;
+import baubles.api.BaubleType;
+import baubles.api.BaublesApi;
+import baubles.api.IBauble;
+import baubles.api.cap.IBaublesItemHandler;
 
-public class ItemRing  extends Item implements IBauble
+@Mod.EventBusSubscriber
+public class ItemRing extends Item implements IBauble
 {
+	public static final Item RING = (new ItemRing()).setUnlocalizedName("Ring").setRegistryName("ring");
 
 	public ItemRing()
 	{
 		super();
 		this.setMaxStackSize(1);
 		this.setHasSubtypes(true);
 		this.setMaxDamage(0);
-		setCreativeTab(CreativeTabs.TOOLS);
+		this.setCreativeTab(CreativeTabs.TOOLS);
+	}
+
+	@SubscribeEvent
+	public static void registerItems(RegistryEvent.Register<Item> event) {
+		event.getRegistry().register(RING);
 	}
 
-	//TODO fix texture
 	@SideOnly(Side.CLIENT)
```

Comment:
as of Forge 14.21.0.2360 this method is no longer client only!
See http://files.minecraftforge.net/maven/net/minecraftforge/forge/1.12-14.21.0.2360/forge-1.12-14.21.0.2360-changelog.txt